mesonbuild/cmake/common.py

```diff
@@ -187,33 +187,33 @@ def __init__(self, data: T.Dict[str, T.Any]) -> None:
                 self.includes += [CMakeInclude(Path(i))]
 
     def log(self) -> None:
-        mlog.log('flags        =', mlog.bold(', '.join(self.flags)))
-        mlog.log('defines      =', mlog.bold(', '.join(self.defines)))
-        mlog.log('includes     =', mlog.bold(', '.join([str(x) for x in self.includes])))
+        mlog.log('flags =', mlog.bold(', '.join(self.flags)))
+        mlog.log('defines =', mlog.bold(', '.join(self.defines)))
+        mlog.log('includes =', mlog.bold(', '.join([str(x) for x in self.includes])))
         mlog.log('is_generated =', mlog.bold('true' if self.is_generated else 'false'))
-        mlog.log('language     =', mlog.bold(self.language))
+        mlog.log('language =', mlog.bold(self.language))
```

Comment:
This section over here is part of the output string, so that has concrete changes to Meson's behavior.

But, it may be fine for the same reason that internal code formatting is fine. :)

Reply:
That's a bit of an oversight on my end. I'll revert that actually, since there are other locations, even within the same file, with this style already. I think we either change them all or leave it as-is.